tools/mason/MasonTest.chpl

```diff
@@ -319,7 +319,7 @@ private proc runTestBinary(projectHome: string, outputLoc: string, testName: str
       erroredTestNames: list(string),
       testsPassed: list(string),
       skippedTestNames: list(string);
-  var localesCountMap: map(int, int, parSafe=true);
+  var localesCountMap: map(int, int, parSafe=false);
```

Comment:
@ben-albrecht Is this adjustment to `localesCountMap` OK? As far as I can tell the map is never used in a parallel context.

Reply:
I think so. Out of curiosity, what is the motivation for updating it?

Reply:
Indexing a map `myMap[foo]` is now illegal for maps initialized with `parSafe=true`.

Reply:
Oh right. Makes sense. There is a desire to run tests in parallel in the future, but we never reached that point in `MasonTest`. When we pick that effort up again, we may decide to convert this to the concurrent form. This seems like the right move for now though. Thanks.